src/js_native_api_v8_internals.h

```diff
@@ -0,0 +1,26 @@
+#ifndef SRC_JS_NATIVE_API_V8_INTERNALS_H_
```

Comment:
I think it would be useful to add some comments at the start of each of the files which document what should/should not go into each file.  This will help both authors and reviewers as new additions are made to the files.  For example, In this file only methods/code which depend directly on host specific APIs (node.js APIs in the Node.js implementation) should be added as opposed to js_native_api_v8.h where functions that depend on v8 but no host specific APIs should go.